x/concentrated-liquidity/keeper_test.go

```diff
@@ -6,28 +6,34 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/stretchr/testify/suite"
 
+	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/internal/math"
+
 	"github.com/osmosis-labs/osmosis/v13/app/apptesting"
 	swaproutertypes "github.com/osmosis-labs/osmosis/v13/x/swaprouter/types"
 )
 
 var (
-	DefaultLowerPrice       = sdk.NewDec(4545)
-	DefaultLowerTick        = int64(84222)
-	DefaultUpperPrice       = sdk.NewDec(5500)
-	DefaultUpperTick        = int64(86129)
-	DefaultCurrPrice        = sdk.NewDec(5000)
-	DefaultCurrTick         = sdk.NewInt(85176)
-	DefaultCurrSqrtPrice, _ = DefaultCurrPrice.ApproxSqrt() // 70.710678118654752440
-	DefaultZeroSwapFee      = sdk.ZeroDec()
-	ETH                     = "eth"
-	DefaultAmt0             = sdk.NewInt(1000000)
-	DefaultAmt0Expected     = sdk.NewInt(998587)
-	USDC                    = "usdc"
-	DefaultAmt1             = sdk.NewInt(5000000000)
-	DefaultAmt1Expected     = sdk.NewInt(4999999999)
-	DefaultLiquidityAmt     = sdk.MustNewDecFromStr("1517818840.967415409394235163")
-	DefaultTickSpacing      = uint64(1)
-	PoolCreationFee         = swaproutertypes.DefaultParams().PoolCreationFee
+	DefaultExponentAtPriceOne      = sdk.NewInt(-4)
+	DefaultMinTick, DefaultMaxTick = math.GetMinAndMaxTicksFromExponentAtPriceOne(DefaultExponentAtPriceOne)
+	DefaultLowerPrice              = sdk.NewDec(4545)
+	DefaultLowerTick               = int64(305450)
+	DefaultUpperPrice              = sdk.NewDec(5500)
+	DefaultUpperTick               = int64(315000)
+	DefaultCurrPrice               = sdk.NewDec(5000)
+	DefaultCurrTick                = sdk.NewInt(310000)
+	DefaultCurrSqrtPrice, _        = DefaultCurrPrice.ApproxSqrt() // 70.710678118654752440
+	DefaultZeroSwapFee             = sdk.ZeroDec()
+	MaxSpotPrice                   = sdk.MustNewDecFromStr("100000000000000000000000000000000000000")
+	MinSpotPrice                   = sdk.MustNewDecFromStr("0.000000000000000001")
```

Comment:
I feel like there were different contexts for this earlier

Reply:
I defined it in constants in the keeper now instead

https://github.com/osmosis-labs/osmosis/pull/3874/commits/84d47e6d3518cab64e0fdfe6ab4de5acbcd8f338